### ISSUE TITLE
Certificate

### ISSUE THREAD

**Issue by Happyfeet01** [nextcloud/social#943]
**Describe the bug**
It seems an Certificate is wrong from mastadon.

**To Reproduce**

**Expected behavior**
No warning message in nextcloud

<details>
<summary>Server details</summary>

**Social app version:**

**Operating system**: 

Ubuntu 18.04 
**Web server:**
nginx
**Database:**

**PHP version:**
7.4
**Nextcloud version:** 
19.0.0

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
[no app in context] Warning: 500 - {"status":-1,"exception":"daita\\MySmallPhpTools\\Exceptions\\RequestNetworkException","message":"51 - SSL: certificate subject name 'scw-blissful-wright' does not match target host name 'social.privacytools.io' - {\"protocols\":[\"https\"],\"used_protocol\":\"https\",\"host\":\"social.privacytools.io\",\"url\":\"\/users\/crimeflare\",\"timeout\":10,\"type\":0,\"data\":[]}"}

POST /apps/social/inbox
from 141.101.69.53 at 2020-07-11T15:01:23+00:00
```

</details>

**Comment by Tachi107** [comment]
I'm getting the same issue, but I would not say that posting to other servers is not possible. You can reply to posts from other servers from your Nextcloud Social instance, but you are required to follow the one who posted because posts from global servers currently appear only in the Home tab. 
The problem only occurs when trying to interact (reply, follow...) with a Mastodon page; it asks you to login, and when filling in your Nextcloud Social's credentials, it reports an error. Maybe that's because Mastodon uses a non standard Mastodon-only way to log in? I don't know, just guessing...

**Comment by Happyfeet01** [comment]
Yes thats is the same experience. You can't also use other android apps to login to your nextcloud social account.

**Comment by spaetz** [comment]
> Yes thats is the same experience. You can't also use other android apps to login to your nextcloud social account.

Well that one would be issue #145 ...

**Comment by Tachi107** [comment]
> Well that one would be issue #145 ...

I agree, this issue should be closed. Misleading title and duplicate of #145

**Comment by pwFoo** [comment]
Maybe related to redirect issue hits the follow feature?
https://github.com/nextcloud/social/issues/630#issuecomment-758888592
